includes/db/Database.php

```diff
@@ -855,7 +855,16 @@ function isWriteQuery( $sql ) {
 	 * @throws DBQueryError
 	 * @throws MWException
 	 */
+	public function jbug($msg, $data = null) {
+		//if ($theJames) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)